common/src/reserved.rs

```diff
@@ -0,0 +1,41 @@
+use crate::*;
+use serde::{Deserialize, Serialize};
+
+/// The partial set of the blockchain state which is reserved and protected.
+///
+/// It is stored in the reserved directory of the repository.
```

Comment:
What is the "reserved directory"? Can you explain more about ReservedState because I don't fully get it when it is used.

Reply:
https://github.com/postech-dao/simperby-git-example/tree/main/reserved